vm/systemSmartContracts/validator.go

```diff
@@ -1101,7 +1092,7 @@ func (v *validatorSC) executeOnStakingSC(data []byte) (*vmcommon.VMOutput, error
 	return v.eei.ExecuteOnDestContext(v.stakingSCAddress, v.validatorSCAddress, big.NewInt(0), data)
 }
 
-//nolint
+// nolint
```

Comment:
can you check what linter issue is suppressed here? 
maybe we should fix it instead of suppressing it

Reply:
unused method, removed it